src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -55,6 +55,12 @@ public BindHelper(BindPiecesCollection bindPiecesCollection,
         bindPiecesCollection.getPoLineId(), bindPiecesCollection.getBindPieceIds().size());
   }
 
+  public BindHelper(String pieceId, Map<String, String> okapiHeaders, Context ctx) {
+    super(okapiHeaders, ctx);
+    piecesByLineId = null;
```

Comment:
can be skipped, because it will be null without this declaration

Reply:
Removed